lib/matplotlib/collections.py

```diff
@@ -2016,6 +2015,57 @@ def set_paths(self):
             self._meshWidth, self._meshHeight, self._coordinates)
         self.stale = True
 
+    def set_array(self, A):
+        if isinstance(self, QuadMesh) and A is not None:
+            width, height = self._meshWidth, self._meshHeight
+            shape = A.shape
+            misshapen_data = False
+            faulty_data = False
+            A_ = A.ravel()
```

Comment:
```suggestion
            misshapen_data = False
            faulty_data = False
```
`A_` is unused, and the usage of `shape` can be replaced by `A.shape` since `A` is not modified. Probably a remainder of a previous state of the code.

Reply:
Oh yeah my bad, sorry I'll remove all of this.